app/controllers/users_controller.rb

```diff
@@ -0,0 +1,4 @@
+class UsersController < ApplicationController
+  def show
+  end
+end
```

Comment:
- [ ] Kindly create an empty index action, like you have created the show action.